.github/workflows/pytorch-latest-release.yml

```diff
@@ -0,0 +1,26 @@
+name: Get latest PyTorch release version
+on:
+  schedule:
+    - cron:  '0 5 * * 1-5'
+jobs:
+  get-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          ref: 'main'
+      - name: Fetch PyTorch release version
+        run: |
+          curl -sL https://api.github.com/repos/pytorch/pytorch/releases/latest | \
+          jq -r ".tag_name" | tr -d 'v' > .github/workflows/pytorch-release-versions/pytorch-latest.txt
+      - name: Check for modified files
+        id: git-check
+        run: echo ::set-output name=modified::$([ -z "`git status --porcelain`" ] && echo "false" || echo "true")
+      - name: Commit latest PyTorch release version
+        if: steps.git-check.outputs.modified == 'true'
+        run: |
+          git config --global user.name 'ClaudiaComito'
+          git config --global user.email 'c.comito@fz-juelich.de@users.noreply.github.com'
+          git commit -am "New PyTorch release"
```

Comment:
Should we add the actual release version here, just to be precise?

Reply:
Done. Also added release version to PR title